### PR TITLE
Change tarball payload for rpm

### PR DESCRIPTION
**Pull request by adamkankovsky**

### CODE REVIEW

File: test/prepare-updates-img [hunk 1]
```diff
@@ -14,6 +14,9 @@ for i in ${TEST_RPMS//,/ }; do
     cp $i tmp/rpms
 done
 
+# Copy the payload RPM to the tmp/rpms
+cp "$(ls bots/images/fedora-rawhide-anaconda-payload.rpm)" tmp/rpms/
```
Comment (KKoukiou) [inline test/prepare-updates-img:18]: This will not read the symlink path: you need: readlink -f bots/images/fedora-rawhide-anaconda-payload

Comment (KKoukiou) [reply test/prepare-updates-img]: @adamkankovsky can you please fix this according to the suggestion? Without readlink we dont get the symlink path

File: test/machine_install.py [hunk 2]
```diff
@@ -83,11 +83,6 @@ def _serve_payload(self):
 
         return payload_cached_name, http_payload_port
 
-    def _write_interactive_defaults_ks(self):
-        payload_cached_name, http_payload_port = self._serve_payload()
```
Comment (KKoukiou) [inline test/machine_install.py:87]: You can also remove the serve_payload function now

File: test/prepare-updates-img (hunk 1 above)
Comment (KKoukiou) [reply test/prepare-updates-img]: Looks like cp -L bots/images/fedora-rawhide-anaconda-payload.rpm tmp/rpms/
works

Comment (adamkankovsky) [reply test/prepare-updates-img]: cp -L unfortunately doesn't work as it should

File: test/prepare-updates-img [hunk 3]
```diff
@@ -14,6 +14,9 @@ for i in ${TEST_RPMS//,/ }; do
     cp $i tmp/rpms
 done
 
+# Copy the payload RPM to the tmp/rpms
+cp "$(readlink -f bots/images/fedora-rawhide-anaconda-payload)" tmp/rpms/
```
Comment (KKoukiou) [inline test/prepare-updates-img:18]: You can also do: `cp -L bots/images/fedora-rawhide-anaconda-payload tmp/rpms/fedora-rawhide-anaconda-payload.rpm`

Feel free top decide, I think the new suggestion is cleaner.

**Comment by KKoukiou** [comment]
Looks like this was not the correct approach https://github.com/cockpit-project/bots/pull/6801

**Pull request closed without merging**